azure/packages/azure-client/src/interfaces.ts

```diff
@@ -121,11 +122,35 @@ export interface AzureContainerServices {
 }
 
 /**
- * Since Azure provides user names for all of its members, we extend the IMember interface to include
- * this service-specific value. It will be returned for all audience members connected to Azure.
+ * Since Azure provides user names for all of its members, we extend the
+ * {@link @fluidframework/protocol-definitions#IUser} interface to include this service-specific value.
+ */
+export interface AzureUser<T = unknown> extends IUser {
```

Comment:
This should be `any`. `any` is used when indicating to the consumer that the type can be anything. `unknown` is used when you want to check against the type before use. For additionalDetails we want to say that you can provide anything and we will proxy it through without caring what it is.

```suggestion
export interface AzureUser<T = any> extends IUser {
```

Reply:
Actually, updating this to use `any` actually ends up violating our linter rules when we attempt to consume it. I would have to add an `as unknown` case in order to satisfy the rule. I think it might be better to leave this alone and to make `AzureMember` use `unknown` as well (though we would have to stage that separately as a breaking change). @skylerjokiel what do you think?

Reply:
Per our offline discussion, I went ahead and updated this to use `any` for now, and updated the type-param docs to note the requirement that input data be JSON-serializable.